lynis/libtofs.jinja

```diff
@@ -0,0 +1,112 @@
+{%- macro files_switch(source_files,
```

Comment:
This is an older version of `libtofs.jinja`.  The latest version is available here:

* https://github.com/saltstack-formulas/template-formula/blob/master/TEMPLATE/libtofs.jinja

Reply:
fixed!